tools/ghpages_documentation_deploy.sh

```diff
@@ -14,7 +14,7 @@
 # GH_TOKEN
 
 # Non-travis variables used by this script.
-TARGET_REPOSITORY_USER="QISKit"
+TARGET_REPOSITORY_USER="Qiskit"
```

Comment:
I'm not sure we can change this, it looks like this a username

Reply:
i think this has changed as we renamed the git org.

Reply:
@diego-plan9 can we change this or do we need to leave this as QISKit?